torch/csrc/Module.cpp

```diff
@@ -1706,6 +1707,7 @@ PyObject* initModule() {
   // into C, so these lines have to execute first)..
   THCPStream_init(module);
   THCPEvent_init(module);
+  THCPGdsFile_init(module);
```

Comment:
Why is this not called from csrc/cuda/Module.cpp ? I would expect all cuda-related initializations to happen there?
If there is no good place there, leaving it here sounds fine.